docs/usage/configuration-options.md

```diff
@@ -1497,7 +1497,17 @@ Use this field to restrict rules to a particular datasource. e.g.
 
 ### matchCurrentVersion
 
-`matchCurrentVersion` can be an exact SemVer version or a SemVer range.
+`matchCurrentVersion` can be an exact SemVer version or a SemVer range:
+
+```json
+{
+  "packageRules": [
+    {
+      "matchCurrentVersion": ">=1.0.0"
```

Comment:
This is an incomplete package rule so not good to use as an example. It should match at least a package name or something like that and then "do something"

Reply:
```suggestion
      "matchCurrentVersion": ">=1.0.0",
      "matchPackageNames": ["angular"]
```